src/main/java/reposense/commits/model/ContributionPair.java

```diff
@@ -0,0 +1,55 @@
+package reposense.commits.model;
+
+/**
+ * Stores the insertions and deletions as a pair.
+ */
+public class ContributionPair {
+    private int insertions;
+    private int deletions;
+
+    public ContributionPair() {
+    }
+
+    public ContributionPair(int insertions, int deletions) {
+        this.insertions = insertions;
+        this.deletions = deletions;
+    }
+
+    public int getInsertions() {
+        return insertions;
+    }
+
+    public int getDeletions() {
+        return deletions;
+    }
+
+    public void setInsertions(int insertions) {
```

Comment:
Is there a need for these setter / getters in the current PR? If not, perhaps we can introduce them when they are needed in future?

Reply:
Removed